### PR TITLE
Lafranch dev

### PR DESCRIPTION
**Pull request by palafranchise**
…roughout the small house.

*Description of changes:*

Adds the route.yaml for this world's map (matches what we're doing in the bookstore world). Makes is easier (or more uniform) to run sample apps (like the robot monitoring sample app) in multiple worlds.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: worlds/small_house.world [hunk 1]
```diff
@@ -22,7 +22,7 @@
     <scene>
       <ambient>0.4 0.4 0.4 1</ambient>
       <background>0.7 0.7 0.7 1</background>
-      <shadows>1</shadows>
```
Comment (samuelgundry) [inline worlds/small_house.world:25]: Is this change intentional?

File: worlds/small_house.world [hunk 2]
```diff
@@ -465,13 +465,13 @@
         <include>
             <uri>model://aws_robomaker_residential_DeskPortraitB_01</uri>
         </include>
-        <pose frame=''>3.547 -5.087950 0.202496 0.007118 -0.000001 3.046683</pose>
```
Comment (samuelgundry) [inline worlds/small_house.world:468]: These changes intentional tool?

**Review by samuelgundry — COMMENTED** [review]
A couple of seemingly unrelated changes?

**Comment by palafranchise** [comment]
Ah, ros1/ros2 branch changes we not limited to mine. Will resubmit.
